data_collection/gazette/spiders/rn_mossoro.py

```diff
@@ -42,3 +38,20 @@ def parse(self, response):
             yield scrapy.Request(
                 f"https://www.dom.mossoro.rn.gov.br{next_page_url}", callback=self.parse
             )
+
+    def parse_gazette_page(self, response, gazette_date):
+        raw_edition_number = response.xpath(
+            '//div[@id="main-content"]//li[3]//strong/text()'
+        ).get()
+        edition_number = re.search(r"\d+", raw_edition_number).group(0)
```

Comment:
O `0` pode ser omitido e diria que até melhora um pouco a legibilidade

```suggestion
        edition_number = re.search(r"\d+", raw_edition_number).group()
```

Reply:
Depois de ver os outputs fui dar uma olhada nas edições com valores repetidos: 83, 98 e 121.

As edições publicadas no fim de semana deveriam ser 83A, 98A e 121A como aparece no site.

Então na verdade não precisamos de regex aqui para identificar apenas os dígitos. O texto do elemento já é o número da edição.

Em paralelo, essas edições que não são apenas dígitos são edições extras.

Reply:
> Em paralelo, essas edições que não são apenas dígitos são edições extras.

Enderecei isso adicionando um novo método `extra_edition_check()`
Ia chamar de `_is_extra_edition` já que retorna um booleano, porém como `is_extra_edition` já é um campo de Gazette, troquei.